examples/MamaDuckVerbose/MamaDuckVerbose.ino

```diff
@@ -37,33 +44,54 @@ void loop() {
   
   //Handle OTA requests
   ArduinoOTA.handle();
+  
+  //Get flag if LoRa packet received
+  if(duck.getFlag()) {
+
+    //Flip lora flag for next message
+    duck.flipFlag(); 
 
-  if(duck.getFlag()) {  //Get flag if LoRa packet received
-    duck.flipFlag(); //Flip lora flag for next message
-    duck.flipInterrupt(); //Flip interrupt to false
-    int pSize = duck.handlePacket(); //Read and store lora packet
+    //Flip interrupt to false
+    duck.flipInterrupt(); 
+
+    //Read and store lora packet
+    int pSize = duck.handlePacket(); 
     Serial.print("runMamaDuck pSize ");
     Serial.println(pSize);
-    if(pSize > 0) { //Make sure the packet has data
-      String msg = duck.getPacketData(pSize); //Decode packet data and store in _lastPacket
-      Packet lastPacket = duck.getPacketData(); //Get _lastPacket data
-      if(msg != "ping" && !duck.idInPath(lastPacket.path)) { //Check if device ID is in path
+
+    //Make sure the packet has data
+    if(pSize > 0) { 
+
+      //Decode packet data and store in _lastPacket
+      String msg = duck.getPacketData(pSize); 
```

Comment:
Why do we need to decode the packet, since we are simply relaying it?
Looking further down I see we have a special case for `ping` message. Why is that?